src/js/components/Ballot/BallotSearch.jsx

```diff
@@ -132,7 +132,7 @@ const SearchWrapper = styled.div`
   border: 1px solid ${props => (!props.searching ? 'rgba(0, 0, 0, 0.23)' : props.brandBlue)};
   padding: 0 3px 0 3px;
   margin-right: 16px;
-  @media (max-width: 960px) {
+  @media (max-width: ${({ theme }) => theme.breakpoints.lg}) {
```

Comment:
Hi @lazorfuzz, I am going to approve this pull request as-is, but over time could you please give these breakpoints full names that match the ones we use for our sass variables? I have learned that short variables over time can cause confusion for engineers doing maintenance on a system, and full variable names that provide more context help a lot. (ex/ instead of "lg", something like "tablet")